nimbus/src/lib.rs

```diff
@@ -313,7 +313,9 @@ pub const SCHEMA_VERSION: u32 = 1;
 pub struct Experiment {
     pub schema_version: String,
     pub slug: String,
-    pub application: String,
+    pub app_name: Option<String>,
+    pub app_id: Option<String>,
+    pub channel: String,
```

Comment:
⚠️  I think this needs a migration! ⚠️

Reply:
channel is now defined as an `Option<String>` and I've added a test to ensure we can deserialize schema variants before and after this change.